index.html

```diff
@@ -252,6 +252,29 @@ <h2 class="heading-2">About <br />Myself</h2>
 
       </ul>
     </section>
-
+    <footer class="background-style">
+      <section class="contact-form" id="contact">
+        <h2 class="contact-title">Contact me</h2>
+        <p class="p-contact">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+        <form class="contact-info" action="https://formspree.io/f/xlevonka" method="POST" id="form">
+          <label for="name"></label>
+          <input type="text" name="Name" class="inp-form" maxlength="30" required placeholder="Name" id="name" />
+          <label for="email"></label>
+          <input type="email" name="_replyTo" class="inp-form" id="email" placeholder="Email" required />
+          <label for="message"></label>
+          <textarea name="message" class="text-area" cols="30" rows="10" maxlength="500"
+            placeholder="Write your message here" required id="message"></textarea>
+          <small id="message-success" class="hidden">From Submitted</small>
+          <input type="submit" class="btncontact" value="Get It Touch" />
+        </form>
+        <div class="contact-bar">
+          <img class="end-line" src="assets/images/Shape.png" alt=" end line icon" />
+        </div>
```

Comment:
- [ ] KIndly remove this line image even if it is visible on the Figma but its role is to indicate that the design is for mobile versions. Kindly remove it.

    - ### Your design
   ![image](https://user-images.githubusercontent.com/87197412/199470632-a0c85f81-4ef3-405c-90f2-95f2a69782c4.png)